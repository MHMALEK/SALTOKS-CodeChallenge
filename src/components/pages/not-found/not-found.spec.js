import React from "react"
import renderer from "react-test-renderer"
import { MemoryRouter } from "react-router-dom"
import NotFound from "./not-found"
describe("Not Found page Snappshot ", () => {
  test("snapshot renders", () => {
    const component = renderer.create(
      <MemoryRouter>
        <NotFound />
      </MemoryRouter>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
