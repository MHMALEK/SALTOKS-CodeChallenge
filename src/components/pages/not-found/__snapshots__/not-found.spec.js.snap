// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found page Snappshot  snapshot renders 1`] = `
<div>
  Not Found
</div>
`;
